src/Microsoft.OData.Edm/Csdl/Serialization/EdmModelCsdlSchemaXmlWriter.Async.cs

```diff
@@ -0,0 +1,1250 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="EdmModelCsdlSchemaXmlWriter.Async.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Xml;
+using Microsoft.OData.Edm.Csdl.CsdlSemantics;
+using Microsoft.OData.Edm.Csdl.Parsing.Ast;
+using Microsoft.OData.Edm.Helpers;
+using Microsoft.OData.Edm.Vocabularies;
+using Microsoft.OData.Edm.Vocabularies.V1;
+
+namespace Microsoft.OData.Edm.Csdl.Serialization
+{
+    /// <summary>
+    /// OData Common Schema Definition Language (CSDL) XML writer

```

Comment:
Across this file: Other than for common/known abbreviations (e.g. CSDL and JSON), please use sentence case in the docs for class, methods, and parameters. Edm or EDM is also okay. References to types (e.g. `EdmRecordExpression`) or method (e.g. `IsType`) are okay too